pkg/services/authz/zanzana.go

```diff
@@ -156,7 +157,7 @@ func (z *Zanzana) running(ctx context.Context) error {
 	if z.cfg.Env == setting.Dev && z.cfg.Zanzana.ListenHTTP {
 		go func() {
 			z.logger.Info("Starting OpenFGA HTTP server")
-			err := zanzana.StartOpenFGAHttpSever(z.cfg, z.handle, z.logger)
+			err := zserver.StartOpenFGAHttpSever(z.cfg, z.handle, z.logger)
```

Comment:
Unrelated to this PR but there is a typo
```suggestion
			err := zserver.StartOpenFGAHttpServer(z.cfg, z.handle, z.logger)
```